### DISCUSSION TITLE
Why does Whipper say that a rip is not accurate, when the file just isn't on the database?

### DISCUSSION THREAD

**Discussion by AggregateAggregate** [whipper-team/whipper#602] [Q&A]
Maybe I'm misunderstanding this... It's just kind of confusing. Whipper says that the rip is not accurate, but the log file just says that the file isn't on the database, but ripped OK.

What does this mean...? This doesn't mean that the rip isn't accurate right?

**Comment by MerlijnWajer** [comment]
That's right, it just means that it cannot verify it to be accurate, I believe.
